docs/user-guide/troubleshooting.md

```diff
@@ -11,60 +11,75 @@ Do not move it without providing redirects.
 
 ## Authentication to the Kubernetes API Server
 
-If your Kubernetes cluster is not configured correctly, it may fail to contact
-the API server. Before
-explaining the solution, it is useful to review how the dashboard discovers
-and authenticates with the API server.
+
 A number of components are involved in the authentication process and the first step is to narrow
 down the source of the problem, namely whether it is a problem with user authentication or with service authentication.
-In the diagram below you can see the full authentication flow, starting with the browser
+Both authentications must work:
+
+```
++-------------+   user             +-------------+   service          +-------------+
+|             |   authentication   |             |   authentication   |             |
+|  browser    +------------------->+  apiserver  +<-------------------+  dashboard  |
+|             |                    |             |                    |             |
++-------------+                    +-------------+                    +-------------+
+
+```
+
+__User authentication__
+
+From your workstation you can connect to Dashboard in three different ways:
+
+1. _Authentication through kubectl:_ This is recommended, because it is secure and easy. In your desktop environment (e.g. laptop)
+verify that kubectl is configured properly with `kubectl cluster-info`. If you get a response then you can continue.
+Enter the command `kubectl proxy`. kubectl will authenticate with the apiserver and proxy every request to a local address which is unprotected. This is not a problem as the site is only accessible locally. Now access Dashboard with `http://localhost:8001/ui`. If it fails then your problem is located with service authentication. See next section.
```

Comment:
authenticate something = check the authenticity of something
'kubectl will authenticate the apiserver' or 'kubectl will use apiserver to authenticate the user' etc...